autopush/endpoint.py

```diff
@@ -649,12 +649,11 @@ def post(self, router_type="", router_token="", uaid="", chid=""):
             return self._write_response(
```

Comment:
Not sure I follow. This is the first instance in POST where we check the router type. The code modification below has router.check_token check token validity, as well as write any response, since each router may have a different method of responding to an invalid token. The only bit that can't be readily determined by the router is the relocation url, which we're building ahead of time and passing. (Few reasons for that, including the check token doesn't care how it was accessed and the URL requires content knowledge router doesn't need.)